src/plugins/vis_type_vega/public/data_model/types.ts

```diff
@@ -115,6 +115,7 @@ export interface OpenSearchDashboards {
   type: string;
   renderer: Renderer;
   visibleVisLayers?: Map<VisLayerTypes, boolean>;
+  visInteractions?: Array<{ event: string; handlerName: string }>;
```

Comment:
nit - can we name and export this type so we don't define it again in `vega_parser`?